### PR TITLE
Update action config docs to v3

### PR DESCRIPTION
**Pull request by uekerman**
Follow up of https://github.com/precice/precice/pull/1614 and https://github.com/precice/precice/pull/1403

Only merge after releasing v3.

### CODE REVIEW

**Comment by BenjaminRodenberg** [comment]
@uekerman Should be ready to merge now that we have `precice-v3`. Right?

**Comment by uekerman** [comment]
> @uekerman Should be ready to merge now that we have `precice-v3`. Right?

Yes, ready for review from my side.

File: pages/docs/configuration/configuration-action.md [hunk 1]
```diff
@@ -51,6 +39,10 @@ All target and source data used in actions require `<read-data ... />` or `<writ
 
 For more details, please refer to the [XML reference](configuration-xml-reference.html).
 
+{% version %}
+In older preCICE versions (before v3.0), much more timings and pre-implemented actions were supported. We removed these as their usefulness was limited.
+{% endversion %}
+
```
Comment (BenjaminRodenberg) [inline pages/docs/configuration/configuration-action.md:45]: Remove this part?

**Review by BenjaminRodenberg — APPROVED** [review]
Minor comment. Rest looks good :+1:

File: pages/docs/configuration/configuration-action.md (hunk 1 above)
Comment (uekerman) [reply pages/docs/configuration/configuration-action.md:45]: I think such hints are good. Users might still come look here. As long as such hints are short.

**Pull request merged**